libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -172,8 +230,18 @@ void AP_Motors6DOF::setup_motors(motor_frame_class frame_class, motor_frame_type
         break;
 
     case SUB_FRAME_CUSTOM:
-        // Put your custom motor setup here
-        //break;
+        // Those motors are just to add them to the list, numbers are useless.
```

Comment:
That is the matrix which provides desired_total_wrench --> thrust_per_motor mapping.

Reply:
numbers are useless ? Why not using the actual numbers then ?
Also, that is only for a specific position of the thrusters, if something change that matrix will change right ?
Finally, how was it calculated ?